pkg/util/senderutil/sender.go

```diff
@@ -29,6 +29,10 @@ func GetSystemUser() *Sender {
 	return &Sender{UserId: "system", Role: constants.RoleGlobalAdmin}
 }
 
+func GetUser(userId, role string) *Sender {
```

Comment:
I think `GetUser(user *pb.User) *Sender` would be better

Reply:
OK, fixed.